src/components/KTable/KTable.vue

```diff
@@ -736,12 +736,20 @@ export default defineComponent({
 
       return `k-table_${Math.floor(Math.random() * 1000)}_${props.fetcherCacheKey}` as string
     })
-    const { query, search } = useDebounce('', 350)
-    const { revalidate } = useRequest(
+
+    const query = ref('')
+    const [searchDebounced, debounceSearch] = useDebounce((q: string) => { query.value = q }, 350)
```

Comment:
These variable names are a bit confusing to me; the names are essentially just flipped

Reply:
Would `debouncedSearch` and `generateDebouncedSearch` sounds better here?